Formula/terraform.rb

```diff
@@ -31,6 +31,10 @@ class Terraform < Formula
     url "https://go.googlesource.com/tools.git", :revision => "977844c7af2aa555048a19d28e9fe6c392e7b8e9"
   end
 
+  # This patch works around a known test failure, which will be fixed in terraform version 0.6.17.
+  # https://github.com/hashicorp/terraform/issues/6709
+  patch :DATA
```

Comment:
Unfortunately it doesn't look like the commit that changes the test value is isolated; it changes other things in the code base: https://github.com/hashicorp/terraform/commit/6a20e8927d52ee4f28dcc0798aa899bec81233dd

I don't know when 0.6.17 will be released. Their release cadence seems to be every few weeks, and 0.6.16 came out on May-9. I didn't see anything in their mailing list about an imminent release. It could be a couple weeks or more before 0.6.17 is released.

There is a fair amount in the 0.6.16 release, which would be nice to have available in homebrew: https://github.com/hashicorp/terraform/blob/master/CHANGELOG.md#0616-may-9-2016

Is there anything I can do to get 0.6.16 merged?